lib/node/runner/api/transaction.go

```diff
@@ -52,28 +52,28 @@ func (api NetworkHandlerAPI) GetTransactionByHashHandler(w http.ResponseWriter,
 	vars := mux.Vars(r)
 	key := vars["id"]
 
-	readFunc := func() (payload interface{}, err error) {
-		found, err := block.ExistsBlockTransaction(api.storage, key)
-		if err != nil {
-			return nil, err
-		}
-		if !found {
-			return nil, errors.BlockTransactionDoesNotExists
-		}
-		bt, err := block.GetBlockTransaction(api.storage, key)
-		if err != nil {
-			return nil, err
-		}
-		payload = resource.NewTransaction(&bt)
-		return payload, nil
+	found, err := block.ExistsBlockTransaction(api.storage, key)
```

Comment:
Instead of `exists` + `get`, why don't you get it directly ?

Reply:
To distinguish error in the database and no error but not exist.

Reply:
[The underlying method already does that](https://github.com/bosnet/sebak/blob/0cc0ebdf63f07b1b2e5dced39ec25b5dfe82dddb/lib/storage/leveldb.go#L136-L162)